app/policies/two_factor_authentication/webauthn_policy.rb

```diff
@@ -34,7 +34,7 @@ def roaming_visible?
     end
 
     def platform_visible?
-      platform_configured? || IdentityConfig.store.platform_auth_set_up_enabled
+      platform_configured?
```

Comment:
Because this was an `||`, I think this should always be `true` now. We could probably get rid of the method altogether?

Reply:
Yes, I think we can get rid of this method and remove the check from the template.

Reply:
I suppose we'd also want to get rid of `roaming_visible?` for that matter.

Reply:
done

Reply:
Did you commit those? I'm not seeing the changes.